frontend/packages/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
what is this needed for? looks like maybe used for testing but should be removed